kml/serdes_gpx.hpp

```diff
@@ -51,6 +52,7 @@ class GpxParser
   GeometryType m_geometryType;
   MultiGeometry m_geometry;
   uint32_t m_color;
+  uint32_t m_globalColor; // To support OSMAnd extensions with single color per GPX file
```

Comment:
How it will be handled differently from m_color?

Reply:
It is not reset in GpxParser::ResetPoint